### PR TITLE
Dogtags - Allow dogtags display to be repositioned

### PR DESCRIPTION
**Pull request by mharis001**
**When merged this pull request will:**
- Allow users to reposition the dogtags onscreen display
- Rename `Dogtag.hpp` to `RscTitles.hpp` and clean up the file
- Preview image:
dogtag_preview

### CODE REVIEW

File: addons/dogtags/CfgUIGrids.hpp [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class CfgUIGrids {
+    class IGUI {
+        class Presets {
+            class Arma3 {
+                class Variables {
+                    grid_ACE_dogtags[] = {{"(safeZoneX + safeZoneW) - (12.9 * ((safeZoneW / safeZoneH) min 1.2) / 40)", "safeZoneY + 0.175 * safeZoneH", "8 * (((safeZoneW / safeZoneH) min 1.2) / 40)", "8 * ((((safeZoneW / safeZoneH) min 1.2) / 1.2) / 25)"}, "(((safeZoneW / safeZoneH) min 1.2) / 40)", "((((safeZoneW / safeZoneH) min 1.2) / 1.2) / 25)"};
```
Comment (commy2) [inline addons/dogtags/CfgUIGrids.hpp]: Suggestion:
`#include "\a3\ui_f\hpp\defineCommonGrids.inc"`
Then this line becomes:
```sqf
GVAR(grid)[] = {
    {
        (safeZoneX + safeZoneW) - 12.9 * GUI_GRID_W,
        safeZoneY + 0.175 * safeZoneH,
        8 * GUI_GRID_W,
        8 * GUI_GRID_H
    },
    GUI_GRID_W,
    GUI_GRID_H
};
```

File: addons/dogtags/RscTitles.hpp [hunk 2]
```diff
@@ -6,34 +6,30 @@ class RscTitles {
...
+        movingEnable = 0;
         class controls {
             class background: RscPicture {
                 idc = 1000;
-                text = QUOTE(PATHTOF(data\dogtagSingle.paa));
-                sizeEx = "(((((safezoneW / safezoneH) min 1.2) / 1.2) / 25) * 1)";
+                x = "profileNamespace getVariable ['IGUI_grid_ACE_dogtags_X', (safeZoneX + safeZoneW) - (12.9 * ((safeZoneW / safeZoneH) min 1.2) / 40)]";
+                y = "profileNamespace getVariable ['IGUI_grid_ACE_dogtags_Y', safeZoneY + 0.175 * safeZoneH]";
+                w = "8 * (((safeZoneW / safeZoneH) min 1.2) / 40)";
+                h = "8 * ((((safeZoneW / safeZoneH) min 1.2) / 1.2) / 25)";
```
Comment (commy2) [inline addons/dogtags/RscTitles.hpp]: And this becomes:
```sqf
x = profileNamespace getVariable ['TRIPLES(IGUI,GVAR(grid),X)', (safeZoneX + safeZoneW) - 12.9 * GUI_GRID_W];
y = profileNamespace getVariable ['TRIPLES(IGUI,GVAR(grid),Y)', safeZoneY + 0.175 * safeZoneH];
w = 8 * GUI_GRID_W;
h = 8 * GUI_GRID_H;
```

File: addons/dogtags/RscTitles.hpp [hunk 3]
```diff
@@ -6,34 +6,30 @@ class RscTitles {
...
                 colorText[] = {1, 1, 1, 1};
                 colorBackground[] = {0, 0, 0, 0};
-                x = ((safezoneX + safezoneW) - (8.4 * (((safezoneW / safezoneH) min 1.2) / 40)) - 2.9 * (((safezoneW / safezoneH) min 1.2) / 40));
-                y = safeZoneY + 0.24 * safezoneH;
-                w = (5.9 * (((safezoneW / safezoneH) min 1.2) / 40));
-                h = (3 * ((((safezoneW / safezoneH) min 1.2) / 1.2) / 25));
+                x = "(1.6 * ((safeZoneW / safeZoneH) min 1.2) / 40) + (profileNamespace getVariable ['IGUI_grid_ACE_dogtags_X', (safeZoneX + safeZoneW) - (12.9 * ((safeZoneW / safeZoneH) min 1.2) / 40)])";
+                y = "(0.065 * safeZoneH) + (profileNamespace getVariable ['IGUI_grid_ACE_dogtags_Y', safeZoneY + 0.175 * safeZoneH])";
+                w = "5.9 * (((safeZoneW / safeZoneH) min 1.2) / 40)";
+                h = "3 * ((((safeZoneW / safeZoneH) min 1.2) / 1.2) / 25)";
```
Comment (commy2) [inline addons/dogtags/RscTitles.hpp]: And this becomes:
```sqf
x = (1.6 * GUI_GRID_W + (profileNamespace getVariable ['TRIPLES(IGUI,GVAR(grid),X)', (safeZoneX + safeZoneW) - GUI_GRID_W]);
y = (0.065 * safeZoneH) + (profileNamespace getVariable ['TRIPLES(IGUI,GVAR(grid),X)', safeZoneY + 0.175 * safeZoneH]);
w = 5.9 * GUI_GRID_W;
h = 3 * GUI_GRID_H;
```

I think this makes it much easier to read and comprehend.

File: addons/dogtags/CfgUIGrids.hpp (hunk 1 above)
Comment (mharis001) [reply addons/dogtags/CfgUIGrids.hpp]: Done

**Pull request merged**